### ISSUE TITLE
EmptyLineBetweenBlocks gets confused by comments

### ISSUE THREAD

**Issue by lencioni** [brigade/scss-lint#716]
With the following configuration:

```yaml
EmptyLineBetweenBlocks:
  enabled: true
```

the following SCSS will report a lint:

```scss
.foo {
  color: red;
  .bar {
    color: blue;
  }
}
```

but the following SCSS does not report a lint:

```scss
.foo {
  color: red; // red
  .bar {
    color: blue;
  }
}
```

same with:

```scss
.foo {
  color: red;
  .bar { // blue
    color: blue;
  }
}
```

same with:

```scss
.foo {
  color: red; 
  // red
  .bar {
    color: blue;
  }
}
```

Also with block style comments `/* */`.

**Comment by fofr** [comment]
@lencioni I came here to raise the same bug.

With the same configuration, the following doesn't report any lints:
```sass
.subsection-description {
  margin-bottom: .9375em;   // 15px
  @include media(tablet) {
    margin-bottom: .625em;  // 20px
  }
}
```
